CONTRIBUTING.md

```diff
@@ -1,55 +1,48 @@
 - [1. Getting Oriented.](#1-getting-oriented)
-  - [1.1. Architecture](#11-architecture)
-  - [1.2. Design](#12-design)
+  - [1.1 General Design and Architecture.](#11-general-design-and-architecture)
+  - [1.2 Users and Stakeholders.](#12-users-and-stakeholders)
+  - [1.3 Specific Dependencies](#13-specific-dependencies)
+  - [1.4 Entity Relationship Diagram (ERD).](#14-entity-relationship-diagram-erd)
 - [2. Machine Setup.](#2-machine-setup)
   - [2.1. Developer Setup and Documentation.](#21-developer-setup-and-documentation)
   - [2.2. Infrastructure Setup and Documentation.](#22-infrastructure-setup-and-documentation)
   - [2.3. Using the devcontainer.](#23-using-the-devcontainer)
 
 ## 1. Getting Oriented.
 
-We're currently working towards our
-[1.0 Andromeda Release](https://github.com/zinc-collective/convene/milestone/1).
-The issues in the milestone are ordered by rough priority.
-
-Each issue labeled `epic` is pretty big. We tend to use checkboxes to decompose
-big issues into smaller pieces of work for people to bite off. We don't expect
-contributors to complete all the work before submitting their first patch.
-Instead, we prefer Contributors to take little bites and submit small patches so
-that we can develop tight, asynchronous feedback loops.
-
-We use labels like `design`, `documentation`, `test`, `code`, `infrastructure`,
-and `security` to indicate what kind of work is remaining on the issue.
+New contributors may want to review
+[the Zinc contribution guide](https://www.zinc.coop/contributing/) to ground themselves in the norms we try to follow across projects.
```

Comment:
I would argue the information in the contribution guide is not as specific as that which is being removed here. \
That said, both are meant to be somewhat general, and the reality is probably a bit more fluid than this doc could provide?

Reply:
I think there's probably some stuff worth keeping that I got rid of (for example the "We don't expect contributors to complete everything for every patch; instead we encourage people to submit pull requests early and ask for guidance and help when they need it."